tests/unit/amo/sagas/test_home.js

```diff
@@ -52,159 +43,55 @@ describe(__filename, () => {
       },
     });
     sagaTester.start(homeSaga);
+
+    const state = sagaTester.getState();
+    baseArgs = { api: state.api };
```

Comment:
Maybe:

```suggestion
    baseApiArgs = { api: state.api };
```

`baseArgs` is a bit too generic.